test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/create-cert.sh

```diff
@@ -12,6 +12,38 @@ cp /otel-collector.crt /otel-collector.key /cfg
 
 chmod 644 /cfg/otel-collector.key
 
+# Generate client certificate for mTLS
```

Comment:
Hey @sandy2008, thanks for picking this work up!

One of the things I really like from #4731 is that it generates all the necessary test certs at build time. The current integration test runs via docker compose and spins up a separate container that runs this `create-cert.sh` script to create a cert. See:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/dd7405654f7349fa6238d16a10146b0df68ee30d/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/docker-compose.yml#L8-L12

Since the certs are created via an MsBuild target now, this container is no longer needed. So these lines from the docker-compose file can be deleted and the `create-cert.sh` file can be deleted, too.

The volume mounts for the other services in the docker compose file will need to be updated to the directory where MsBuild put the certs.

Reply:
Thank you for pointing that out! 😊 I’ll update the docker-compose.yml by removing the create-cert service and adjusting the volume mounts to the directory where MsBuild places the certs. The create-cert.sh script will also be removed. I appreciate your help in streamlining the integration tests! 🙌

Reply:
Pushed, please review :) @alanwest 
 ✔ Container opentelemetry-dotnet-tests-1           Stopped                0.1s 
 ✔ Container opentelemetry-dotnet-otel-collector-1  Stopped                0.0s